synfig-studio/src/gui/dialogs/dialog_setup.cpp

```diff
@@ -904,6 +914,12 @@ Dialog_Setup::on_autobackup_changed()
 	auto_backup_interval.set_sensitive(toggle_autobackup.get_active());
 }
 
+void
+Dialog_Setup::on_number_of_core_select()
+{
+	ThreadPool::instance().set_num_threads(int(adj_number_of_core->get_value()));
```

Comment:
Gtk/gtkmm has a method here that help you: `Gtk::SpinButton::get_value_as_int()` :D

Reply:
get_value_as_int() shows error as it is not a member of Gtk::Adjustment
in other places also it is not used
https://github.com/synfig/synfig/blob/master/synfig-studio/src/gui/dialogs/dialog_setup.cpp#L760

Reply:
It's for `Gtk::SpinButton`, not for its `Gtk::Adjustment`. But ok.